mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1158,6 +1159,7 @@ def _evaluate_custom_metrics(self, eval_df):
                     else custom_metric.name
                 )
                 self.metrics_values.update({name: metric_value})
+            _logger.info(f"Finished evaluating custom metric {name}")
```

Comment:
is this line useful? users would see "Evaluating..." at the start and then "Finished..." followed by another "Evaluating..." immediately.

fwiw i don't see other usages of "Finished..." logging